migrating_from_ocp_3_to_4/index.adoc

```diff
@@ -0,0 +1,86 @@
+=== Planning network considerations
+Before migrating from {product-title} 3 to 4, review the following information to xref:../migrating_from_ocp_3_to_4/planning-migration-3-4.adoc#migration-preparing-networking[networking considerations]:
+
+** DNS considerations
+*** Isolating the DNS domain of the target cluster from the clients.
+*** Setting up the target cluster to accept the source DNS domain.
+
+You can migrate stateful application workloads from {product-title} 3 to 4 at the granularity of a namespace. To learn more about MTC see xref:../migrating_from_ocp_3_to_4/about-mtc-3-4.adoc#about-mtc-3-4[understanding MTC].
+
+[NOTE]
+====
+If you are migrating from {product-title} 3, see xref:../migrating_from_ocp_3_to_4/about-migrating-from-3-to-4.adoc#about-migrating-from-3-to-4[About migrating from {product-title} 3 to 4] and xref:../migrating_from_ocp_3_to_4/installing-3-4.adoc#migration-installing-legacy-operator_installing-3-4[Installing the legacy {mtc-full} Operator on {product-title} 3].
+====
+
+[id="mtc-overview-install-mtc"]
+== Installing MTC
+Review the following tasks to install the MTC:
+
+. Install the {mtc-full} Operator on target cluster by using Operator Lifecycle Manager (OLM).
+* {product-title} 4.6 and later versions: xref:../migrating_from_ocp_3_to_4/installing-3-4.adoc#installing-3-4[Install the Migration Toolkit for Containers Operator by using OLM].
+. Install the legacy {mtc-full} Operator on source cluster manually.
+. xref:../migrating_from_ocp_3_to_4/installing-3-4.adoc#configuring-replication-repository_installing-3-4[Configure object storage to use as a replication repository].
+
+[id="mtc-overview-upgrade-mtc"]
+== Upgrading MTC
+You can xref:../migrating_from_ocp_3_to_4/upgrading-3-4.adoc#upgrading-3-4[upgrade the {mtc-full} ({mtc-short})] on {product-title} {product-version} by using OLM. You can upgrade {mtc-short} on {product-title} 3 by reinstalling the legacy {mtc-full} Operator.
+
+[id="mtc-overview-mtc-checklists"]
+== Reviewing MTC checklists
+Before you migrate your application workloads with the Migration Toolkit for Containers (MTC), review the xref:../migrating_from_ocp_3_to_4/premigration-checklists-3-4.adoc#premigration-checklists-3-4[premigration checklists].
+
+[id="mtc-overview-migrate-mtc-applications"]
+== Migrating applications
+You can migrate your applications by using the MTC xref:../migrating_from_ocp_3_to_4/migrating-applications-3-4.adoc#migrating-applications-mtc-web-console_migrating-applications-3-4[web console] or xref:../migrating_from_ocp_3_to_4/advanced-migration-options-3-4.adoc#migrating-applications-cli_advanced-migration-options-3-4[the command line].
+
+[id="mtc-overview-advanced-migration-options"]
+== Advanced migration options
+You can automate your migrations and modify the `MigPlan` and `MigrationController` custom resources in order to perform large-scale migrations and to improve performance. You can check the following items:
```

Comment:
I don't think we need to specify which CRs we  can change. It's a bit too much detail for an overview.
"the `MigPlan` and `MigrationController` custom resources" > "MTC custom resources"

Checking the following items is not quite the right wording because these are options, not checklists.

Maybe this would work: "You can automate migrations and improve the performance of large-scale migrations by using the following options:"

Reply:
acked